source/_integrations/air_quality.markdown

```diff
@@ -2,6 +2,7 @@
 title: Air Quality
 description: Instructions on how to add air quality sensors with Home Assistant
 ha_release: 0.85
+ha_iot_class: Cloud Polling
```

Comment:
Why is this cloud polling?

Reply:
I'm sorry that's my mistake. Would it be local polling instead then?

Reply:
I was just wondering how you came to this conclusion?

Reply:
I mistakenly thought that the integration might connect to an online service from reading just this page https://www.home-assistant.io/integrations/air_quality/

Reply:
These changes we are doing here is not about guessing. If you don't know, it might be a better idea to skip it.

```suggestion
ha_iot_class: ~
```

This is an entity component. It doesn't do anything by itself, it is here for other integration to build upon. Hence it doesn't communicate or provide data by itself. And should therefore be set to null/none.